config/src/config/network_config.rs

```diff
@@ -305,6 +305,22 @@ impl NetworkConfig {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct NetworkServiceConfig {
+    pub max_concurrent_requests: u64, // Max num of concurrent server tasks
+    pub max_network_channel_size: u64, // Max num of pending network messages
+}
```

Comment:
I'm wondering what is this config, and how does it differ from the NetworkConfig?  This can be very confusing to have "NetworkConfig" and "NetworkServiceConfig"

Reply:
This is the config for the service specifically and not the network stack. It'll grow as things get more complicated, so it makes sense to keep them separate. I think better naming will also help reduce confusion (see below) 😄